### ISSUE TITLE
Call for Maintainers

### ISSUE THREAD

**Issue by ethagnawl** [tmuxinator/tmuxinator#884]
I have been the primary maintainer of this library for the last few years but I no longer have the bandwidth to do so.

I'm planning to cut one final release (3.1.0) and then turn maintenance back over to @tmuxinator/tmuxinator. It'd also be great to onboard new folks who'd be interested in contributing. I'm not going to disappear entirely and abandon the project but I can't promise that I'll be triaging bug reports, cutting releases, etc. with any sort of regularity going forward.

**Comment by dysnomian** [comment]
I've been looking for an OSS project to get involved with and I love tmuxinator! I'll email you.

**Comment by akofink** [comment]
I'd be happy to get involved. I've been using tmuxinator for about 12 years, and I still use it every day. I'd like to see the project continue to improve incrementally and maintain support for newer releases of Ruby and tmux.

**Comment by ethagnawl** [comment]
@geoffreyvanwyk Yes. A number of wonderful folks have gotten involved since this issue was initially opened and have taken the reins. If you'd like to get involved, please connect with @akofink.
